pkgs/top-level/all-packages.nix

```diff
@@ -24699,6 +24699,8 @@ with pkgs;
 
   rapidjson = callPackage ../development/libraries/rapidjson { };
 
+  rapidjson-git = callPackage ../development/libraries/rapidjson/git.nix { };
```

Comment:
We usually call these packages `unstable`, so I think this should be `rapidjson-unstable`.